test/test.js

```diff
@@ -31,6 +31,9 @@ describe("csv-loader", function () {
   it("parses csv with the recommended configuration", function (done) {
     var compiler = webpack(DEFAULT_CONFIG,
       function (err, stats) {
+        if (err) {
+          done(err);
```

Comment:
Does the test fail in this case?

Reply:
TBH for some weird reason there was a timeout error happening and I saw that it was un-handled, so I thought I will add this just in case.